Formula/php.rb

```diff
@@ -2,9 +2,8 @@ class Php < Formula
   desc "General-purpose scripting language"
   homepage "https://www.php.net/"
   # Should only be updated if the new version is announced on the homepage, https://www.php.net/
-  url "https://www.php.net/distributions/php-7.4.7.tar.xz"
-  mirror "https://fossies.org/linux/www/php-7.4.7.tar.xz"
```

Comment:
https://fossies.org/linux/www/php-7.4.8.tar.xz

Reply:
Or rather, please don't remove the mirror.

Reply:
Done. For the future: Shall I keep the mirror even if the mirror hasn't been updated yet? It did not have the file when I opened the PR.

Reply:
yeah, keep the mirror, it might be slow ftp server update.